doc/source/glossary.rst

```diff
@@ -99,35 +102,41 @@ Glossary
 
 
    along an axis
-       Reversing along axis 0 (the row axis) below reverses the columns, and
-       reversing along axis 1 (the column axis) reverses the rows. This might
-       be the opposite of what a new user would expect.
+       An operation `along axis n` of array ``a`` behaves as if its argument
```

Comment:
Using backticks for emphasis probably isn't a good idea: numpydoc will try to match them against links wherever possible, using italic text only as a fallback.

Rst syntax for italics is `*text*`.

Reply:
Ah nevermind, this refers to the glossary heading so is fine